content/projects/data-science-specific/data-visualisation/mobile-money-viz/_index.md

```diff
@@ -23,11 +23,11 @@ tags:
 title: Financial Services Use in Tanzania
 ---
 
-## Background
+## About the DataSet
```

Comment:
Why is DataSet capitalised like that?

It is either a "dataset" or a "data set". 

You can call it "DataSet" if it's a class name.